File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_29.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

public class MB_S_29 {

    /// Аббревиатура типа задания на списание
    @JsonProperty("TASK_TYPE")
    @Getter private String taskType;

    /// Вид движения (управление запасами)
    @JsonProperty("BWART")
    @Getter private String bwart;

    /// Место возникновения затрат
    @JsonProperty("KOSTL")
    @Getter private String kostl;

    /// Склад
    @JsonProperty("LGORTTO")
    @Getter private String lgortto;

    /// Требуется отправка в ГИС
    @JsonProperty("SEND_GIS")
    @Getter private String sendGis;

    /// Признак, что для данного типа задания выбирать причину списания не нужно
    @JsonProperty("NO_GRUND")
    @Getter private String noGrund;

    /// Полное название типа задания на списание
    @JsonProperty("LONG_NAME")
    @Getter private String longName;

    /// Лимит количества товаров в задании
    @JsonProperty("LIMIT")
    @Getter private double limit;

    /// Используется в производстве
    @JsonProperty("CHK_OWNPR")
    @Getter private String chkOwnpr;

    public MB_S_29(String taskType, String bwart, String kostl, String lgortto, String sendGis, String noGrund, String longName, double limit, String chkOwnpr) {
        this.taskType = taskType;
        this.bwart = bwart;
        this.kostl = kostl;
        this.lgortto = lgortto;
        this.sendGis = sendGis;
        this.noGrund = noGrund;
        this.longName = longName;
        this.limit = limit;
        this.chkOwnpr = chkOwnpr;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_29.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

class MB_S_29(
        /// Аббревиатура типа задания на списание
        @SerializedName("TASK_TYPE")
        val taskType: String,

        /// Вид движения (управление запасами)
        @SerializedName("BWART")
        val bwart: String,

        /// Место возникновения затрат
        @SerializedName("KOSTL")
        val kostl: String,

        /// Склад
        @SerializedName("LGORTTO")
        val lgortto: String,

        /// Требуется отправка в ГИС
        @SerializedName("SEND_GIS")
        val sendGis: String,

        /// Признак, что для данного типа задания выбирать причину списания не нужно
        @SerializedName("NO_GRUND")
        val noGrund: String,

        /// Полное название типа задания на списание
        @SerializedName("LONG_NAME")
        val longName: String,

        /// Лимит количества товаров в задании
        @SerializedName("LIMIT")
        val limit: Double,

        /// Используется в производстве
        @SerializedName("CHK_OWNPR")
        val chkOwnpr: String
)